### ISSUE TITLE
Difference in PPO's value_loss scale between clipped and non-clipped version

### ISSUE THREAD

**Issue by lucasb-eyer** [ikostrikov/pytorch-a2c-ppo-acktr#135]
Hi Ilya, nice code 👍 

Just wanted to mention a little detail, as in the title. In the clipped version you include the factor of 0.5 in the loss, but in the non-clipped version, you call PyTorch's `mse_loss` which [does not include the 0.5 factor](https://pytorch.org/docs/stable/_modules/torch/nn/functional.html#mse_loss).

So the meaning of `value_loss_coef` depends (by a factor 0.5) on the value of `use_clipped_value_loss`, which makes comparing experiments/tuning hyperparameters less clear. It's a minor difference and you might not want to change it now that many people are using your code, but I wanted to mention it.

**Comment by ikostrikov** [comment]
Hi Lucas!

Thanks a lot! 

Fixed in https://github.com/ikostrikov/pytorch-a2c-ppo-acktr/commit/12c39346f789d714c4a5cd793cc6266757616d80
